doc/fluid/design/concepts/cpp_data_feeding.md

```diff
@@ -153,13 +153,17 @@ double_buffer_reader = create_double_buffer_op(batch_reader)
 The forwarding ops of the corresponding `main_program` would be like this:
 
 ```
-while_op {
+not_completed = true
+pass_count = 0
```

Comment:
`pass_count` is a Python variable, not a ProgramDesc Var, does it work?

Reply:
`not_completed` and `pass_count` are both ProgramDesc Var. Sorry I didn't make it clear.